src/containers/App.jsx

```diff
@@ -32,21 +32,35 @@ import { Konnector } from 'components/Konnector'
 import DefaultRedirectionSnackbar from 'components/DefaultRedirectionSnackbar/DefaultRedirectionSnackbar'
 import ReloadFocus from './ReloadFocus'
 import FooterLogo from 'components/FooterLogo/FooterLogo'
-import useCustomShortcuts from 'components/Shortcuts/useCustomShortcuts'
 
-const IDLE = 'idle'
-const FETCHING_CONTEXT = 'FETCHING_CONTEXT'
+import {
+  mkHomeMagicFolderConn,
+  mkHomeCustomShorcutsConn,
+  mkHomeCustomShorcutsDirConn,
+  contextQuery
+} from 'queries'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
 
 window.flag = window.flag || flag
 window.minilog = minilog
 
+const formatShortcuts = (folders, shortcuts) => {
```

Comment:
nit: It could be extract into a helpers file

Reply:
Oh yes, I should have used the already existing method. Forgot about this one. Thanks.